manim/mobject/geometry.py

```diff
@@ -809,3 +782,51 @@ class RoundedRectangle(Rectangle):
     def __init__(self, **kwargs):
         Rectangle.__init__(self, **kwargs)
         self.round_corners(self.corner_radius)
+
+
+class ArrowTip(VMobject):
+    CONFIG = {
+        "fill_opacity": 0,
+        "stroke_width": 0,
+        "length": DEFAULT_ARROW_TIP_LENGTH,
+        "start_angle": PI,
+    }
+
+    def get_base(self):
+        return self.point_from_proportion(0.5)
+
+    def get_tip_point(self):
+        return self.points[0]
+
+    def get_vector(self):
+        return self.get_tip_point() - self.get_base()
+
+    def get_angle(self):
+        return angle_of_vector(self.get_vector())
+
+    def get_length(self):
+        return get_norm(self.get_vector())
+
+
+class ArrowTriangleTip(ArrowTip, Triangle):
```

Comment:
Comment: I'm usually leery of multiple inheritance, but seeing as these classes are (a) very simple and (b) not meant to be inherited from, then I _guess_ it's ok.

Reply:
I'm with you on being cautious regarding multiple inheritance. In this case, on the one hand, I believe it is actually semantically correct -- and on the other hand, I needed it for `ArrowCircleTip`: without having this class inherit from `Circle`, it would not populate `self.points`.

(But sure, I guess if it were necessary to remove multiple inheritance, a workaround could be found.)

Reply:
Yeah, agreed. Perhaps we can just leave a line somewhere in the docs recommending that people not inherit from them.